src/components/book/book.js

```diff
@@ -0,0 +1,13 @@
+const Book = (book, key) => {
```

Comment:
- [ ] <b>REQUIRED:</b> As I mentioned in the previous review, you did not validate these two props in this component. Kindly validate them by using propTypes;